hbase2/src/test/java/site/ycsb/db/hbase2/HBaseClient2Test.java

```diff
@@ -209,5 +256,68 @@ public void testUpdate() throws Exception{
   public void testDelete() {
     fail("Not yet implemented");
   }
+
+  private void testScanWithValueFiltering(String operation, String filterValue, int scanRowLimit,
+                                          byte[][] expectedValuesReturned) throws Exception {
+    Properties properties = new Properties();
+    properties.setProperty("hbase.usescanvaluefiltering", String.valueOf(true));
+    if(operation != null) {
+      properties.setProperty("hbase.scanfilteroperator", operation);
+    }
+    if(filterValue != null) {
+      properties.setProperty("hbase.scanfiltervalue", filterValue);
+    }
+
+    // setup the client and fill two columns with data
+    setUp(properties);
+    setupTableColumnWithHexValues("col_1");
+    setupTableColumnWithHexValues("col_2");
+
+    Vector<HashMap<String, ByteIterator>> result = new Vector<>();
+
+    // first scan the whole table (both columns)
+    client.scan(tableName, "00000", scanRowLimit, null, result);
+
+    assertEquals(expectedValuesReturned.length, result.size());
+    for(int i = 0; i < expectedValuesReturned.length; i++) {
+      final HashMap<String, ByteIterator> row = result.get(i);
+      assertEquals(2, row.size());
+      assertTrue(row.containsKey("col_1") && row.containsKey("col_2"));
+      assertArrayEquals(expectedValuesReturned[i], row.get("col_1").toArray());
+      assertArrayEquals(expectedValuesReturned[i], row.get("col_2").toArray());
+    }
+
+    // now scan only a single column (the filter should work here too)
+    result = new Vector<>();
+    client.scan(tableName, "00000", scanRowLimit, ImmutableSet.of("col_1"), result);
```

Comment:
Instead of this `ImmutableSet` import use the jdk provided `Collections.singleton`